pkg/middleware/rbac.go

```diff
@@ -35,6 +35,7 @@ func NewRbac(config Rbac) echo.MiddlewareFunc {
 	}
 	return func(next echo.HandlerFunc) echo.HandlerFunc {
 		return func(c echo.Context) error {
+			logger := zerolog.Ctx(c.Request().Context())
```

Comment:
Logging works with the request id for me. However, when making a request without the `x-Rh-Insights-Request-Id` header, RBAC errors are logged with the random request id generated by `echo_middleware.RequestIDWithConfig`. This is in contrast to logging in other locations, such as when a task is enqueued/dequeued/finished. In those log messages, the request id is empty when not provided in the request. What is the intended behavior for requests missing the `x-Rh-Insights-Request-Id` header?

Reply:
I missed this, thanks. I've updated it so now the request ID, in any place, should use the generated one if a request ID is not specified.